cmd/di.go

```diff
@@ -180,6 +182,12 @@ func (di *Dependencies) Bootstrap(nodeOptions node.Options) error {
 	log.Infof("Starting Mysterium Node (%s)", metadata.VersionAsString())
 	log.Infof("Build information (%s)", metadata.BuildAsString())
 
+	// check early for presence of an already running node
+	listener, err := net.Listen("tcp", fmt.Sprintf("%s:%d", nodeOptions.TequilapiAddress, nodeOptions.TequilapiPort))
```

Comment:
`Bootstrap()` is not to run things, just to register components.
You can move listener run to beginning of `httpAPIServer.StartServing()` - it's 1st action we run

Reply:
That's not going to work. I need to check for presence of an already running myst BEFORE initializing the storage.